jest/setup.js

```diff
@@ -1,4 +1,5 @@
 import Enzyme from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
+import 'whatwg-fetch';
```

Comment:
We may want to include `whatwg-fetch` inside the webpack conf instead of importing it here.

Reply:
I'll open an issue therefore.